src/tests/060-tripTest.js

```diff
@@ -3,7 +3,7 @@ import chaiHttp from 'chai-http';
 import dotenv from 'dotenv';
 import app from '../index';
 import tripMockData from './mock/tripMockData';
-import mockData from './mock/mockData';
+import mockData, { superAdminToken } from './mock/mockData';
```

Comment:
Eric, can you help us refactor this, superAdminToken is in mockData.

Reply:
Done.